src/EditorFeatures/Core/Preview/AbstractPreviewFactoryService.cs

```diff
@@ -701,12 +701,6 @@ private async ValueTask<ITextBuffer> CreateTextBufferCoreAsync(TextDocument docu
             rightWorkspace = null;
         };
 
-        if (_editorOptionsService.GlobalOptions.GetOption(SolutionCrawlerRegistrationService.EnableSolutionCrawler))
```

Comment:
note: this was not a user visible option.  We had it because i guess we just like options.  Note: if this was ever something turned off it woudl break all diagnostics as well as unit testing.  As an unsupported and undocumented internal option, it has just been removed.

Reply:
> We had it because i guess we just like options

This was primarily added to allow us to turn off solution crawler in RPS.